http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -188,21 +190,22 @@ abstract class LeoRoutes(
     ExceptionHandler {
       case requestValidationError: RequestValidationError =>
         complete(StatusCodes.BadRequest, requestValidationError.getMessage)
+      case e: ClusterNotFoundException =>
```

Comment:
Just curious, why add this? I think `ClusterNotFoundException` is an instance of `LeoException`

Reply:
I was suspicious that this error was causing 500, so tried making it explicit this is returning 400...But it might've been a misunderstanding..I can remove this for now